packages/console/src/command-history/CommandHistory.test.jsx

```diff
@@ -38,176 +46,192 @@ function mountItems(items) {
+function dragRange(start, end) {
+  const startItem = screen.getByText(`Command ${start}`);
+  fireEvent.mouseDown(startItem);
   for (let i = start; i <= end; i += 1) {
-    getItem(wrapper, i).simulate('mousemove', mouseEventInit);
+    const item = screen.getByText(`Command ${i}`);
+    fireEvent.mouseMove(item);
   }
-  getItem(wrapper, end).simulate('mouseup', mouseEventInit);
+  const endItem = screen.getByText(`Command ${start}`);
+  fireEvent.mouseUp(endItem);
 }
 
 it('renders an empty list without crashing', () => {
-  mountItems([]);
+  mountItems(0);
 });
 
 it('renders a list with items without crashing', () => {
-  mountItems(defaultItems);
+  mountItems();
 });
 
-it('handles selecting an item on click', () => {
-  const wrapper = mountItems(defaultItems);
+function expectSelected(index) {
+  expect(screen.getByText(`Command ${index}`).parentNode.className).toContain(
+    'active'
+  );
+}
+function expectNotSelected(index) {
+  expect(
+    screen.getByText(`Command ${index}`).parentNode.className
+  ).not.toContain('active');
+}
 
-  clickItem(wrapper, 0);
+it('handles selecting an item on click', () => {
+  mountItems();
 
-  expect(getItem(wrapper, 0).hasClass('selected')).toBe(true);
-  expect(getItem(wrapper, 1).hasClass('selected')).toBe(false);
+  clickItem(0);
+  expectSelected(0);
 });
 
 it('handles selecting and deselecting an item on click', () => {
-  const wrapper = mountItems(defaultItems);
+  mountItems(10);
 
-  clickItem(wrapper, 0);
+  userEvent.click(screen.getByText('Command 0'));
 
-  expect(getItem(wrapper, 0).hasClass('selected')).toBe(true);
-  expect(getItem(wrapper, 1).hasClass('selected')).toBe(false);
+  expectSelected(0);
+  expectNotSelected(1);
 
-  clickItem(wrapper, 0);
+  userEvent.click(screen.getByText('Command 0'));
 
-  expect(getItem(wrapper, 0).hasClass('selected')).toBe(false);
-  expect(getItem(wrapper, 1).hasClass('selected')).toBe(false);
+  expectNotSelected(0);
+  expectNotSelected(1);
 });
 
 it('handles changing selection on click', () => {
-  const wrapper = mountItems(defaultItems);
+  mountItems(10);
 
-  clickItem(wrapper, 0);
+  clickItem(0);
 
-  expect(getItem(wrapper, 0).hasClass('selected')).toBe(true);
-  expect(getItem(wrapper, 1).hasClass('selected')).toBe(false);
+  expectSelected(0);
+  expectNotSelected(1);
 
-  clickItem(wrapper, 1);
+  clickItem(1);
 
-  expect(getItem(wrapper, 0).hasClass('selected')).toBe(false);
-  expect(getItem(wrapper, 1).hasClass('selected')).toBe(true);
+  expectSelected(1);
+  expectNotSelected(0);
 });
 
 it('handles selecting a range on shift click', () => {
-  const wrapper = mountItems(defaultItems);
+  mountItems();
 
-  clickItem(wrapper, 3);
-
-  expect(getItem(wrapper, 2).hasClass('selected')).toBe(false);
-  expect(getItem(wrapper, 3).hasClass('selected')).toBe(true);
-  expect(getItem(wrapper, 4).hasClass('selected')).toBe(false);
-
-  clickItem(wrapper, 7, { shiftKey: true });
+  clickItem(3, { shiftKey: true });
```

Comment:
Shouldn't need the shift on the first click:
```suggestion
  clickItem(3);
```